artifacts-guidance.md

```diff
@@ -1,6 +1,29 @@
 # Guidance for Artifacts Authors
 
-Content other than OCI container images MAY be packaged using the image manifest.
-When this is done, the `config.mediaType` value should not be a known OCI image config [media type](media-types.md).
-Historically, due to registry limitations, some tools have created non-OCI conformant artifacts using the `application/vnd.oci.image.config.v1+json` value for `config.mediaType` and values specific to the artifact in `layer[*].mediaType`.
+## Artifacts and Images
+
+This specification is primarily concerned with packaging two kinds of content: Artifacts, and Images. Both are representing using a [manifest](manifest.md).
+Images are defined in this specification as conformant content with a [config](config.md), designed to be interpreted by a runtime that implements the [runtime-spec][]. 
+Conversely, an Artifact is any other conformant content that **does not contain a config to be interpreted by a runtime-spec implementation.**
+
+## Creating an Artifact
+
+Content other than Images MAY be packaged using the [manifest]; this is otherwise known as an Artifact.
+When this is done, the `artifactType` should be set to a custom media type, or the `config.mediaType` should not be a known Image config [media type](media-types.md).
 Implementation details and examples are provided in the [image manifest specification](manifest.md#guidelines-for-artifact-usage).
+
+Note: Historically, due to registry limitations, some tools have created non-conformant Artifacts using the `application/vnd.oci.image.config.v1+json` value for `config.mediaType`.
```

Comment:
IMO we really can't get away from specifying that "images" have strict layer media type requirements too -- if a given runtime can't parse one of the layers, it can't reasonably assume it can (safely) do anything with the rest of that registry object (manifest, artifact, image, whatever you want to call it).

Reply:
I don't want to address that in this PR yet. As a follow-up to this, I want us to discuss rootfs vs non-rootfs layer types and how we can consistently decide which layers must be interpreted and which are "optional."